services/ymax-planner/src/support.ts

```diff
@@ -46,7 +46,12 @@ export const usdcAddresses: UsdcAddresses = {
 // Avg measurements of Actual Gas used for EVM Txs
 export const gasLimitEstimates = {
   Factory: 1_209_435n, // https://sepolia.arbiscan.io/tx/0xfdb38b1680c10919b7ff360b21703e349b74eac585f15c70ba9733c7ddaccfe6
-  Wallet: 276_809n,
+  Supply_Aave: 279_473n, // ref. https://github.com/Agoric/agoric-sdk/issues/12021#issuecomment-3361285596
+  Supply_Compound: 151_692n,
+  Withdraw_Aave: 234_327n,
+  Withdraw_Compound: 123_081n,
+  DepositForBurn: 151_320n,
+  Wallet_Fallback: 276_809n,
```

Comment:
lets keep similar formatting for all.
```suggestion
  Supply_Aave: 279_473n, // ref. https://github.com/Agoric/agoric-sdk/issues/12021#issuecomment-3361285596
  SupplyCompound: 151_692n,
  WithdrawAave: 234_327n,
  WithdrawCompound: 123_081n,
  DepositForBurn: 151_320n,
  WalletFallback: 276_809n,
```